### PR TITLE
US001 - Selenium UI test project has now been added, 1 UI test implemented

### PR DESCRIPTION
**Pull request by liammc98**
Added a new Selenium test project so that we can run UI tests against the website.
1 UI test has been implemented to check if the page is loaded/initialised. The test passes.

### CODE REVIEW

**Review by RyanCunning1 — APPROVED** [review]

File: Views/src/test/java/JobsListTest.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import pages.JobsListPage;
+
+import static org.junit.Assert.assertTrue;
+
+public class JobsListTest extends BaseTest {
+
+    @Test
+    public void initializeJobsListPage() {
+        // go to this page
+        driver.get("http://localhost:7999/JobsList");
```
Comment (John-Neill-Kainos) [inline Views/src/test/java/JobsListTest.java:12]: Is this link not meant to be http://localhost:7999/jobslist ? Just wondering

Comment (liammc98) [reply Views/src/test/java/JobsListTest.java]: Updated now

Comment (John-Neill-Kainos) [reply Views/src/test/java/JobsListTest.java]: okay cool, I just wasnt sure if the caps would actually work or not ....

**Review by jkelso662 — APPROVED** [review]
Looks good to me. Looking forward to seeing these running!

**Review by John-Neill-Kainos — APPROVED** [review]

**Pull request merged**